myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -0,0 +1,376 @@
+        score = [self._get_score(float(self.obs_dict['time'])) if win_cdt else 0][0]
+
+        rwd_dict = collections.OrderedDict((
+                # Optional Keys
+                ('act_reg', act_mag),
+                # Must keys
+                ('sparse',  score),
+                ('solved',    win_cdt),
+                ('drop', lose_cdt),
+                ('done',  self._get_done()),
+            ))
+        rwd_dict['dense'] = np.sum([wt*rwd_dict[key] for key, wt in self.rwd_keys_wt.items()], axis=0)
+        return rwd_dict
+
+    def get_metrics(self, paths):
+        """
+        Evaluate paths and report metrics
+        """
+        # average sucess over entire env horizon
+        score = np.mean([np.sum(p['env_infos']['rwd_dict']['sparse']) for p in paths])
+        points = np.mean([np.sum(p['env_infos']['rwd_dict']['solved']) for p in paths])
+        times = np.mean([np.round(p['env_infos']['obs_dict']['time'][-1],2) for p in paths])
+        # average activations over entire trajectory (can be shorter than horizon, if done) realized
+
+        metrics = {
+            'score': score,
+            'points': points,
+            'times': times,
+            }
+        return metrics
+
+    def step(self, *args, **kwargs):
+        self._update_opponent_state()
+        obs, reward, done, info = super().step(*args, **kwargs)
+        return obs, reward, done, info
+
+    def reset(self):
+        self._sample_opponent_policy()
+        if self.reset_type == 'random':
+            qpos, qvel = self._get_randomized_initial_state()
+        elif self.reset_type == 'init':
+                qpos, qvel = self.sim.model.key_qpos[2], self.sim.model.key_qvel[2]
+        else:
+            qpos, qvel = self.sim.model.key_qpos[0], self.sim.model.key_qvel[0]
+        qpos, qvel = self._initialize_opponent(qpos, qvel)
+        self.robot.sync_sims(self.sim, self.sim_obsd)
+        obs = super(WalkEnvV0, self).reset(reset_qpos=qpos, reset_qvel=qvel)
+        return obs
+
+    def viewer_setup(self, *args, **kwargs):
+       """
+       Setup the default camera
+       """
+       distance = 5.0
+       azimuth = 90
+       elevation = -15
+       lookat = None
+       self.sim.renderer.set_free_camera_settings(
+               distance=distance,
+               azimuth=azimuth,
+               elevation=elevation,
+               lookat=lookat
+       )
+       render_tendon = True
+       render_actuator = True
+       self.sim.renderer.set_viewer_settings(
+           render_actuator=render_actuator,
+           render_tendon=render_tendon
+       )
+
+    def set_action_normalization(self, flag: bool):
+        """
+        Activates and deactivates action remapping.
+        :param flag:
+            "1" means: [-1, 1] -> [0, 1] via a non-linear remapping function. See envs/myo/base_v0.py
+            "0" means: [-1, 1] -> [0, 1] via simple clipping.
+        """
+        self.normalize_act = flag
```

Comment:
This can be configured during env registration. Is it helpful to have another function? 
Be careful that hot-swapping this flag works well with all initializations.